### PR TITLE
[DFMD-4] Add Page Navigation to your Pages

### PR DESCRIPTION
**Pull request by Teecols**
IMG_20210304_095911_090

### CODE REVIEW

**Comment by Teecols** [comment]
IMG_20210304_095917_654

File: submissions/tosinKolawole/html-forms/product-page.html [hunk 1]
```diff
@@ -0,0 +1,49 @@
+﻿<!DOCTYPE html>
```
Comment (adaugochi) [inline submissions/tosinKolawole/html-forms/product-page.html:1]: You don't have t create a `product-page.html` file inside the `html-forms` folder. Kindly link the `product-page.html` that is in the `html` folder

**Review by adaugochi — CHANGES_REQUESTED** [review]

File: submissions/tosinKolawole/html-forms/product-page.html (hunk 1 above)
Comment (Teecols) [reply submissions/tosinKolawole/html-forms/product-page.html:1]: I don't really understand what I'm supposed to do. Can you kindly expatiate

Comment (adaugochi) [reply submissions/tosinKolawole/html-forms/product-page.html:1]: During your `html` task, you created a `product page` file and you placed that file in your `html` folder I guess. For this task, you still went ahead to create the same `product-page.html` file in your `html-forms` folder. This wrong. You don't have to create another one in your `html-forms` folder. Your product page should be located in your `html` folder not anywhere else.

**Review by adaugochi — CHANGES_REQUESTED** [review]

File: submissions/tosinKolawole/html-forms/Contactus.html [hunk 2]
```diff
@@ -5,6 +5,22 @@
     <title>Contact Form</title>
 </head>
 <body>
+    <div>
+        <ul>
+           <li><a href="html-forms/product-page.html">Product</a></li> 
+           <li><a href="html-forms/contact-form.html">Contact us</a></li><br>
```
Comment (adaugochi) [inline submissions/tosinKolawole/html-forms/Contactus.html:11]: You have rename your file from `contact-form.html` to `Contactus.html`

**Review by adaugochi — CHANGES_REQUESTED** [review]

**Comment by adaugochi** [comment]
You still haven't take correction

**Comment by Teecols** [comment]
In your last comment, you said I changed the name of the Contact-form.html to Contactus.html. As a result, I changed it back to the previous name. Kindly specify the exact correction you're referring to so that I can effect it immediately.

**Comment by Ireti203** [comment]
Remove the Homepage.html file from the html-forms folder. You had earlier created a product page, place the navigation links there.

**Comment by adaugochi** [comment]
Unattended to

**Pull request closed without merging**